src/runners/cairoRunner.test.ts

```diff
@@ -29,12 +34,20 @@ describe('cairoRunner', () => {
   describe('runUntilPc', () => {
     test('should return the value of the 10th fibonacci number', () => {
       const runner = new CairoRunner(PROGRAM);
-      const finalPc = new Relocatable(0, 12);
-      runner.runUntilPc(finalPc, false);
+      runner.runUntilPc(runner.finalPc, false, false);
       const executionSize = runner.vm.memory.getSegmentSize(1);
       const executionEnd = runner.executionBase.add(executionSize);
 
       expect(runner.vm.memory.get(executionEnd.sub(1))).toEqual(new Felt(144n));
     });
+
+    test('should export encoded trace and memory', () => {
+      const runner = new CairoRunner(PROGRAM);
+      runner.runUntilPc(runner.finalPc, true, true);
+      const trace_filename = 'fibonacci_trace_ts.bin';
+      const memory_filename = 'fibonacci_memory_ts.bin';
+      runner.exportTrace(path.join(tmpDir, trace_filename));
+      runner.exportMemory(path.join(tmpDir, memory_filename));
+    });
```

Comment:
so this is 2 tests in one 1 test, plz split

+, you should at least assert that the file exists, and better also test the content of the file. We could run this test with a simpler cairo program ?